webapi/WebapiBuild.sbt

```diff
@@ -162,17 +163,17 @@ lazy val webApiLibs = Seq(
 
 lazy val javaRunOptions = Seq(
     // "-showversion",
-    "-Xms2048m",
-    "-Xmx4096m"
+    "-Xms8G",
```

Comment:
Are you sure you need this much memory? This seems to be making the Travis build fail, because the JVM can't allocate that much memory on Travis.